src/Root.h

```diff
@@ -212,10 +212,10 @@ class cRoot
 		cCommandOutputCallback * m_Output;
 	} ;
 
-	typedef std::map<AString, cWorld *> WorldMap;
+	typedef std::map<AString, cWorld> WorldMap;
 	typedef std::vector<cCommand> cCommandQueue;
 
-	cWorld * m_pDefaultWorld;
+	AString m_DefaultWorldName;
```

Comment:
What's the point of this change?

Reply:
haha, good pointer joke :D

...I've forgotten. Maybe to make unambiguous that it can't be null? I'll try to change it back and see if I hit any blocks.

Reply:
Ok! Reverted with apparently no ill effects.